docs/test_frameworks/xunit_net.md

```diff
@@ -1,8 +1,14 @@
 # Testing with xUnit.net
 
-The [Testcontainers.Xunit](https://www.nuget.org/packages/Testcontainers.Xunit) package simplifies writing tests with containers in [xUnit.net](https://xunit.net). By leveraging xUnit.net's [shared context](https://xunit.net/docs/shared-context), this package automates the setup and teardown of test resources, creating and disposing of containers as needed. This reduces repetitive code and avoids common patterns that developers would otherwise need to implement repeatedly.
+The [Testcontainers.Xunit](https://www.nuget.org/packages/Testcontainers.Xunit) and [Testcontainers.XunitV3](https://www.nuget.org/packages/Testcontainers.XunitV3) packages simplify writing tests with containers in [xUnit.net](https://xunit.net). By leveraging xUnit.net's [shared context](https://xunit.net/docs/shared-context), this package automates the setup and teardown of test resources, creating and disposing of containers as needed. This reduces repetitive code and avoids common patterns that developers would otherwise need to implement repeatedly.
 
-To get started, add the following dependency to your project file:
+To get started with xUnit.net v3, add the following dependency to your project file:
+
+```shell title="NuGet"
+dotnet add package Testcontainers.XunitV3
+```
+
+If you use xUnit.net v2, add the following dependency to your project file:
```

Comment:
~~~suggestion
To get started, add the following dependency to your project file for xUnit.net v3:

```shell title="NuGet"
dotnet add package Testcontainers.XunitV3
```

Or, if you're using xUnit.net v2:
~~~